src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/edit/details/main.js

```diff
@@ -163,7 +163,7 @@ define([
                     '/admin/api/media/' + curMediaId + '?action=new-version' :
                     '/admin/api/media?collection=' + this.formOptions.accountAvatarCollection;
                 
-                url = url + '&locale=' + encodeURIComponent(this.sandbox.sulu.user.locale);
+                url = url + '&locale=' + encodeURIComponent(this.sandbox.sulu.getDefaultContentLocale());
```

Comment:
Just returning the first content locale is probably not a good solution here... Maybe we should also take the content language into account? /cc @chirimoya